docs/support/index.md

```diff
@@ -1,4 +1,4 @@
-# Help
+# Support
```

Comment:
@LalithVipulananthan I also updated the title.

Reply:
Thank you. I was planning to do that with my next PR, but on reflection it makes sense to do it now since the "menu" is being updated in this PR to change "Help" to "Support".